website/blog/2022/04/29/interview with ju zhiyuan: becoming one of the 918 asf members, i felt thrilled and proud

```diff
@@ -0,0 +1,181 @@
+---
+title: "Interview with JU Zhiyuan: Becoming One of the 918 ASF Members, I Felt Thrilled and Proud"
+slug: /blog/2022/04/29/interview-juzhiyuan-apache-member
+authors:
+  - name: "Yahui Rao"
```

Comment:
Not sure if you want to go with Estelle Rao or Yahui Rao.
```suggestion
  - name: "Estelle Rao"
```

Reply:
I prefer using Estelle Rao.